CMSgov/easi-app
# [EASI-2733, EASI-2734] - TRB Admin request home

# EASI-2733
# EASI-2734

[Figma](https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?node-id=193-15620&t=faiQWVsgvgNLjFpi-0)

## Changes and Description

- View for TRB Admin Home
- InformationCard component for rendering TRB forms and docs summary
- Added typePolicy config for trbRequest query entities with no `id`

Outstanding work: 
Unit test

![Screenshot 2023-03-29 at 2 23 23 PM](https://user-images.githubusercontent.com/95709965/228632571-c011a208-6fee-4884-bf26-4c1943480745.png)

## How to test this change

- Create a TRB request
- Verify all states of consult, TRB Lead, initial request form, and documents are all reflected accurately

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
